### PR TITLE
LLM: Add Replit CPU and GPU example

### PR DESCRIPTION
**Pull request by JinBridger**
## Description

Add Replit CPU and GPU example in `python/llm/example` using `replit/replit-code-v1-3b`

### CODE REVIEW

**Review by plusbang — COMMENTED** [review]
Besides, the example layout is updated, please rebase and move to new folders.

File: python/llm/example/gpu/hf-transformers-models/replit/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+For optimal performance on Arc, it is recommended to set several environment variables.
+
+```bash
+export USE_XETLA=OFF
+export SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS=1
+```
+
+```
+python ./generate.py --repo-id-or-model-path REPO_ID_OR_MODEL_PATH --prompt PROMPT --n-predict N_PREDICT
```
[the remark this answers is not in the document]
Comment (JinBridger) [reply python/llm/example/gpu/hf-transformers-models/replit/README.md]: Got it. I've rewritten it.

File: python/llm/example/transformers/transformers_int4/replit/README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+```bash
+conda create -n llm python=3.9
+conda activate llm
+
+pip install bigdl-llm[all] # install bigdl-llm with 'all' option
+```
+
+### 2. Run
+```
+python ./generate.py --repo-id-or-model-path REPO_ID_OR_MODEL_PATH --prompt PROMPT --n-predict N_PREDICT
```
[the remark this answers is not in the document]
Comment (JinBridger) [reply python/llm/example/transformers/transformers_int4/replit/README.md]: Got it. I've rewritten it.

**Review by plusbang — COMMENTED** [review]
Please update links in https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/CPU/HF-Transformers-AutoModels/Model#verified-models and https://github.com/intel-analytics/BigDL/tree/main/python/llm/example/GPU/HF-Transformers-AutoModels/Model#verified-models, other LGTM

**Review by plusbang — APPROVED** [review]
LGTM

**Pull request merged**